chalk-ir/src/visit.rs

```diff
@@ -180,6 +188,29 @@ where
         Self::Result::new()
     }
 
+    fn visit_free_var_const(
```

Comment:
At the moment, we could merge all these `visit_free_var_*` functions into a single `visit_free_var` function, as we don't handle them differently (and the same with `visit_bound_var_*` functions). Though maybe these might be useful in the future? Not sure.

Reply:
Hmm, good point. I agree it's not clear if anything in the future would ever handle these differently. Perhaps not.